ext/afform/admin/ang/afGuiEditor/afGuiSearch.html

```diff
@@ -23,7 +23,7 @@
             </li>
           </ul>
         </div>
-        <input ng-if="filter.mode === 'routeParams'" class="form-control" ng-model="filter.value" />
+        <input ng-if="filter.mode === 'routeParams'" class="form-control" ng-model="filter.value" title="Append '#/?{{ filter.value }}=value' to URL to pass parameter value to form." />
```

Comment:
I'm not up to date on the recommended place to put these kind of tips, but just commenting from a technical point of view it should be translatable by using the `ts` function (see e.g. line 22 above). But this one would be a trickier because some of it is literal code, so it would be something like (untested) `{{:: ts('Append %1 to the URL to pass a parameter value to the form' , {1: "'#/?{{ filter.value }}=value'"}) }}`. May need to fiddle with the brackets and any escaping.

Reply:
Thanks so much for the comments and suggestion. Took a bit of fiddling but have added the translateable function now https://github.com/bmango/civicrm-core/commit/b91474ec07dcec977f6e5f15c1795ce21f2952d1.

Reply:
The end result was `title="{{:: ts('Append %1 to the URL to pass a parameter value to the form' , {1: '\'#/?' + filter.value + '=value\''}) }}"`